src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/jms/JmsErrorNotificationSender.java

```diff
@@ -32,6 +40,14 @@ public JmsErrorNotificationSender(
         this.containerMappings = containerMappings;
     }
 
+    /**
+     * Sends an error notification to the error notifications queue.
+     * @param zipFilename The name of the zip file
+     * @param containerName The name of the container
+     * @param eventId The event ID
+     * @param errorCode The error code
+     * @param cause The exception that caused the error
```

Comment:
Everything is in method signature order apart from the cause

Reply:
fixed